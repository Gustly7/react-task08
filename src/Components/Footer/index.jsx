import React, { Component } from 'react'

class Footer extends Component {
  render () {
    return (
      <div className='Footer'>
        <span className='title red-font'>netflixroulette</span>
      </div>
    )
  }
}

export default Footer
