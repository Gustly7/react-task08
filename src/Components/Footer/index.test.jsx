import React, { Component } from 'react'
import Footer from './index'
import { shallow } from 'enzyme'
import renderer from 'react-test-renderer'

describe('Footer component test suite', () => {
  test('Footer snapshot renders correctly', () => {
    const tree = renderer.create(
      <Footer />
    ).toJSON()
    expect(tree).toMatchSnapshot()
  })

  test('Footer div should have "Footer" class', () => {
    const wrapper = shallow(
      <Footer />
    )
    const elem = wrapper.find('div.Footer')
    expect(elem.text()).toBe('netflixroulette')
  })
})
