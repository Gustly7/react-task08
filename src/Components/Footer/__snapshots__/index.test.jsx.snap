// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component test suite Footer snapshot renders correctly 1`] = `
<div
  className="Footer"
>
  <span
    className="title red-font"
  >
    netflixroulette
  </span>
</div>
`;
